card/mod/05_standard/set/self/signin.rb

```diff
@@ -96,6 +95,10 @@ def default_core_args args={}
   end  
 end
 
+event :signin_success, :after=>:signin do
+  abort :success
+end
+
```

Comment:
Oog, I can't remember how the ordering works.  I've really got to get on the documentation work (planning to send something out about that soon. I have a rough plan in mind, but this is not a great place to discuss that)

I guess the latter has the advantage that you don't have to remember the ordering logic ;)